src/simulation/m_start_up.fpp

```diff
@@ -165,7 +167,15 @@ contains
             k_x, k_y, k_z, w_x, w_y, w_z, p_x, p_y, p_z, &
             g_x, g_y, g_z, n_start, t_save, t_stop, &
             cfl_adap_dt, cfl_const_dt, cfl_target, &
-            viscous, surface_tension
+            viscous, surface_tension, &
+            lag_bubbles, lag_write_bubbles, lag_write_bubble_stats, &
+            lag_bubble_model, lag_cluster_type, lag_heatTransfer_model,&
+            lag_massTransfer_model, lag_nBubs_glb, csonhost, vischost, &
+            Thost, gammagas, gammavapor, pvap, cpgas, cpvapor, &
+            kgas, kvapor, Rgas, Rvap, diffcoefvap, sigmabubble, &
+            lag_rkck_tolerance, lag_smooth_type, lag_epsilonb, &
```

Comment:
`lag_XXXX` variables (except `lag_bubbles`) should be put into a derived type in my view so they are all encapsulated to one input variable. also makes for clearer code. so the derived type should be, for example `EL%wite_bubbles`, `EL%write_bubble_stats`, etc.